postgres/datadog_checks/postgres/postgres.py

```diff
@@ -72,6 +74,52 @@ def __init__(self, name, init_config, instances):
         self._db_pool = {}
         self._db_pool_lock = threading.Lock()
 
+        self.tags = copy.copy(self._config.tags)
+        self.tags=[t for t in self.tags if not t.startswith("db:")]
```

Comment:
```suggestion
        self.tags_without_db=[t for t in copy.copy(self._config.tags) if not t.startswith("db:")]
```
We seem to filter out the `db:` tag in a few areas, perhaps we can clean it up with this.
Besides updating your changes to use this new field, these can be replaced as well:
- https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/postgres.py#L106
- https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/postgres.py#L292
- https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/postgres.py#L347